gallery/src/components/NxDrawer/NxDrawerPage.tsx

```diff
@@ -0,0 +1,258 @@
+        <NxP>
+          The <NxCode>NxDrawer</NxCode> component implements a dialog sidebar that is fixed to the
+          right side of the <NxCode>NxPage</NxCode> layout content. It consists of three
+          parts: <NxCode>NxDrawer.Header</NxCode> (required), <NxCode>NxDrawer.Content</NxCode> (required),
+          {' '}<NxCode>NxFooter</NxCode> (optional). The <NxCode>NxDrawer</NxCode> is only compatible
+          within <NxCode>NxPage</NxCode> layout with global header (it is not supported in legacy layout).
+        </NxP>
+        <NxTile.Subsection>
+          <NxH3>Props</NxH3>
+          <NxTable>
+            <NxTable.Head>
+              <NxTable.Row>
+                <NxTable.Cell>Prop</NxTable.Cell>
+                <NxTable.Cell>Type</NxTable.Cell>
+                <NxTable.Cell>Required</NxTable.Cell>
+                <NxTable.Cell>Default</NxTable.Cell>
+                <NxTable.Cell>Details</NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Head>
+            <NxTable.Body>
+              <NxTable.Row>
+                <NxTable.Cell>className</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  Any <NxCode>className</NxCode> attributes passed in on
+                  the <NxCode>NxDrawer</NxCode> will be added to
+                  the dialog element.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>onCancel</NxTable.Cell>
+                <NxTable.Cell>Function (() =&gt; void)</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  <NxCode>NxDrawer</NxCode>'s callback function. It gets called when the user
+                  closes the dialog (when the user clicks outside the drawer, when the user clicks the close button,
+                  and when the user presses the Esc key).
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>variant</NxTable.Cell>
+                <NxTable.Cell>"narrow" | "normal"</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>"normal"</NxTable.Cell>
+                <NxTable.Cell>
+                  This prop specifies a style variant for the drawer. Currently, variants only differ in width.
+                  "normal" drawers are 520px wide, and "narrow" drawers are 348px wide.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>HTML <NxCode>&lt;dialog&gt;</NxCode> Attributes</NxTable.Cell>
+                <NxTable.Cell>
+                  <NxTextLink external href="https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes">
```

Comment:
This isn't the link we usually use for this.  I'd use https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog for this case.

Reply:
> This isn't the link we usually use for this. I'd use https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog for this case.

Updated here: 2998f1692a0e07c855d2ea6e936bca97d53515e7